drivers/bridge/bridge.go

```diff
@@ -782,7 +782,9 @@ func (d *driver) deleteNetwork(nid string) error {
 			logrus.Warn(err)
 		}
 		if link, err := d.nlh.LinkByName(ep.srcName); err == nil {
-			d.nlh.LinkDel(link)
+			if err := d.nlh.LinkDel(link); err != nil {
+				logrus.WithError(err).Errorf("Failed to delete interface (%q)'s link on endpoint (%q) delete", ep.srcName, ep.id)
```

Comment:
sorry probably I was not clear in the previous comment, what I was suggesting is that if you use the formatter %q you can remove the brackets.
See here the difference: https://play.golang.com/p/HBUIUuAx7UE
`(%q)` seems too much